.github/actions/check_test_project_fixture/check_test_project_fixture.mjs

```diff
@@ -47,7 +47,7 @@ if (hasFixtureOkLabel) {
           'That usually means the test-project fixture needs to be rebuilt.',
           `If you know that it doesn't, add the "fixture-ok" label. Otherwise, rebuild the fixture and commit the changes:`,
           '',
-          '  yarn build:test-project --rebuild-fixture',
+          '  yarn build:test-project-rebuild',
```

Comment:
@Tobbe heads up that this needs to be updated

Reply:
Thanks!
https://github.com/redwoodjs/redwood/pull/8624